jenkins/jobs/netbsd6-smoke.yml

```diff
@@ -0,0 +1,39 @@
+- job:
+    name: netbsd6-smoke
+    node: netbsd_build
+    description: GlusterFS pre-commit smoke test on NetBSD
+    project-type: freestyle
+
+    scm:
+    - glusterfs
+
+    triggers:
+    - gerrit:
+        trigger-on:
+          - patchset-created-event:
+              exclude-drafts: false
+              exclude-trivial-rebase: false
+              exclude-no-code-change: true
+          - draft-published-event
+          - comment-added-contains-event:
+              comment-contains-value: "recheck smoke"
+        server-name: review.gluster.org_for-smoke-jobs
+        projects:
+          - project-compare-type: 'PLAIN'
+            project-pattern: 'glusterfs'
+            branches:
+              - branch-compare-type: 'ANT'
+                branch-pattern: '**'
+
+    parameters:
+    - string:
+        default: refs/heads/master
+        description: 'For review 12345 and patch set 6, this will be refs/changes/45/12345/6. Default: Tip of master'
+        name: GERRIT_REFSPEC
+    - string:
+        default: master
+        description: 'Name of the branch you want to build from. We usually build from master'
+        name: GERRIT_BRANCH
+
+    builders:
+    - shell: /opt/qa/build.sh
```

Comment:
1. So, I've been moving more things to do `!include-raw`
2. This job literally just runs `/opt/qa/build.sh`. There didn't seem to be a need to write a separate script and then include it here.